eth/watchers/roundswatcher.go

```
@@ -75,6 +88,10 @@ func (rw *RoundsWatcher) Watch() error {
 	}
 	rw.setLastInitializedRound(lr, bh)
 
+	if err := rw.fetchAndSetTranscoderPoolSize(); err != nil {
+		glog.Errorf("error fetching initial transcoderPoolSize: %v", err)
```

Since we're returning an error on L83 and L87 let's also return an error here.